src/OneExplorer/ConfigObject.ts

```diff
@@ -0,0 +1,194 @@
+ * ```
+ *
+ * @usage Create Parsed Config Object
+ *        (use a factory function `createConfigObj`)
+ * ```
+ * const configObj = createConfigObj(uri);
+ * ```
+ */
+export class ConfigObj {
+  /**
+   * source uri of config file
+   */
+  uri: vscode.Uri;
+
+  /**
+   * a raw ini object read from config file
+   */
+  rawObj: object;
+
+  /**
+   * a parsed config object
+   */
+  obj: {
+    baseModels: vscode.Uri[],
+    derivedModels: vscode.Uri[],
+    // TODO: Add more fields
+  };
+
+  private constructor(uri: vscode.Uri, rawObj: object) {
+    this.uri = uri;
+    this.rawObj = rawObj;
+    this.obj = {
+      baseModels: ConfigObj.parseBaseModels(uri, rawObj),
+      derivedModels: ConfigObj.parseDerivedModels(uri, rawObj)
+    };
+  }
+
+  /**
+   * A simple parse function with directly `ConfigObj.obj` fields after parsing
+   * without keeping an ConfigObj instance.
+   *
+   * @param uri
+   * @returns `{baseModels, derivedModels}` or
+   *          `null` if failed to open/parse into ini object
+  }
+   */
+  public static parse(uri: vscode.Uri): {baseModels: vscode.Uri[],
+                                         derivedModels: vscode.Uri[]}|null {
+    const cfgObj = this.createConfigObj(uri);
+
+    if (!cfgObj) {
+      return null;
+    }
+
+    return cfgObj.obj;
+  }
+
+  /**
+   * A factory function to create ConfigObj for the given uri.
+   *
+   * @param uri
+   * @returns `ConfObj` or
+   *          `null`    if failed to open/parse into ini object
+   */
+  public static createConfigObj(uri: vscode.Uri) {
+    const obj = this.importIni(uri.fsPath);
+    if (!obj) {
+      console.error(`Cannot open ${uri.fsPath}`);
+
+      return null;
+    }
+
+    return new ConfigObj(uri, obj);
+  }
+
+  /**
+   * Import an ini file
+   * @param filePath
+   * @returns `object` if file read is successful, or `null` if file open has failed
+   *
+   */
+  private static importIni(filePath: string): object|null {
+    let configRaw: string;
+    try {
+      configRaw = fs.readFileSync(filePath, 'utf-8');
+    } catch (e) {
+      console.error(e);
+      return null;
+    }
+
+    // TODO check if toString() is required
+    return ini.parse(configRaw.toString());
+  }
+
+  /**
+   * Parse base models written in the ini object and return the absolute path.
+   *
+   * ABOUT MULTIPLE BASE MODELS
+   * onecc doesn't support multiple base models.
+   * However, OneExplorer will show the config node below multiple base models
+   * to prevent a case that users cannot find their faulty config files on ONE explorer.
+   */
+  private static parseBaseModels = (uri: vscode.Uri, iniObj: object): vscode.Uri[] => {
+     const baseModels = ['tflite','pb','onnx']
+     .filter(ext=>iniObj[`one-import-${ext}` as keyof typeof iniObj] ?.['input_path'] !== undefined)
+     .map(ext => iniObj[`one-import-${ext}` as keyof typeof iniObj] ?.['input_path'])
+     .map(relpath=>path.join(path.dirname(uri.fsPath),relpath))
```

Comment:
From the example in https://github.com/Samsung/ONE-vscode/issues/644#issuecomment-1131374528,
- cfg uri.fspath = /home/eric/m/src/model/onecc-test/cfg/inception_v3_conv2d.0to128.cfg
- relpath = ./inception_v3/2020-01-31/inception_v3_conv2d.0to128.tflite

It seems that joining cfg  uri.fspath and relpath in L135 produces non-existing path. 😿 

If we need to assume some constraint regarding path of cfg and base model, could you write that as comment?

Reply:
Oh sure. I can use RealPath to resolve the problem. Thanks for finding that out.

Reply:
@hyunsik-yoon I examined the point and found out that your `OneccRunner` runs onecc on `config file's directory`.
I think that's why I didn't confront an error while testing the current PR.

https://github.com/Samsung/ONE-vscode/blob/5c3c1f636c6ac115b2c8fb46d49bf85fd66c660d/src/OneExplorer/OneExplorer.ts#L477

So it resolves the problem about cwd problem, and it seems okay.

Reply:
Directory issue is currently under discussion in other issue, so I guess we'd better pass the issue this time. 😸